test/Microsoft.IdentityModel.Tokens.Tests/Microsoft.IdentityModel.Tokens.Tests.csproj

```diff
@@ -24,6 +24,10 @@
     <PackageReference Include="Microsoft.Azure.KeyVault.Cryptography" Version="$(MicrosoftAzureKeyVaultCryptographyVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'net461'">
```

Comment:
Every test target, except .NET FWK 4.6.1 supports the `Microsoft.Extensions.TimeProvider.Testing`.